### PR TITLE
adds first draft for a layout page

### PR DESCRIPTION
**Pull request by ewatch**

### CODE REVIEW

File: src/pages/basic-concepts/layouts.md
Comment (FredKSchott) [inline src/pages/basic-concepts/layouts.md:94]: remove this and all below?

**Review by FredKSchott — APPROVED** [review]
LGTM, content-wise! If anyone has style suggestions, feel free to add. Otherwise, I'd be +1 to merge this and then come back again later once all the docs pages are filled in, and do one big style/editing/grammar pass.

**Review by Hanawa02 — COMMENTED** [review]
Added some small comments to it, after fixed should be good to be merged.

**Comment by FredKSchott** [comment]
This is one of the last docs remaining, Merging and addressing these comments!

File: src/pages/basic-concepts/layouts.md
Comment (FredKSchott) [reply src/pages/basic-concepts/layouts.md:94]: nvm, didn't realize that this was a code block

**Pull request merged**
